Projects/1-Beginner/Calculator-App/Calculator-App.py

```
@@ -0,0 +1,89 @@
+        self.main.mainloop()
+
+    def CreateFrames(self):
+        self.mainFrame = ttk.Frame(self.main, height=400, width=300)
+        self.mainFrame.pack(padx=5, pady=5, fill="both", expand=True)
+        self.screenFrame = ttk.Frame(self.mainFrame)
+        self.screenFrame.pack(padx=0, pady=0, expand=True)
+        self.buttonsFrame = ttk.Frame(self.mainFrame)
+        self.buttonsFrame.pack(padx=0, pady=0, expand=True)
+
+    def GridButtons(self):
+        l= 0
```

Ruff (0.12.2)

Ambiguous variable name: `l`

(E741)